cmd/appliance/upgrade/complete.go

```diff
@@ -266,6 +268,8 @@ func upgradeCompleteRun(cmd *cobra.Command, args []string, opts *upgradeComplete
 					appliances = append(appliances[:i], appliances[i+1:]...)
 				}
 			}
+		} else if result.Status == appliancepkg.UpgradeStatusSuccess && rebootRegex.MatchString(result.Details) {
```

Comment:
Why do we need to match the details here?
The success status should be enough to know that we need to issue the upgrade switch partition command.

This couples the logic unnecessarily to the wording of the details IMO.

Reply:
Well, it depends on weather there can really be any other message when the appliance is in a success state? If not, then yes, this is not neccessary.

Reply:
No but there is no other purpose for the success state, so why check the message at all?

Reply:
This prevents us from rewording if e.g. Jamie finds some better wording from an English ,anguage perspective.

Reply:
now it's removed